plugins/modules/monitoring/airbrake_deployment.py

```diff
@@ -103,6 +117,7 @@ def main():
             user=dict(required=False),
             repo=dict(required=False),
             revision=dict(required=False),
+            version=dict(required=False),
```

Comment:
```suggestion
            version=dict(required=False, type='str'),
```

Reply:
Should types be added to the other params?

Reply:
Yes please. But feel free to do it in another PR.

Reply:
Cool. I've got the branch ready to submit a new PR once this one is merged.